sdk/eventhub/.azure-pipelines/pr.client.yml

```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT THIS FILE
+# This file is/may be generated automatically and any changes may be lost.
+
+
+pr:
+- master
+
+trigger:
+  paths:
+    include:
+    - sdk/eventhub/
```

Comment:
While these paths are OK for now we will have to think about if we want to scope to just data-plane once we move over the management libraries under the same directory.

Reply:
Yep ... I think we'll end up with ```pr.client.yml``` referencing a set of paths, and if the ADX team wants to do something similar they might have a ```pr.mgmt.yml``` with another set, possibly with some overlap if there are common dependencies.